resources/authgear/templates/en/web/authflow_reset_password_success.html

```diff
@@ -7,6 +7,10 @@ <h1 class="m-0 primary-txt text-center text-xl font-bold">{{ template "reset-pas
 
 <div class="text-sm break-words primary-txt text-center">{{ template "reset-password-success-description" }}</div>
 
+{{ if .CanBackToSignIn }}
+<a class="as-btn not-a btn primary-btn mt-5" href="{{ call $.MakeURL "/login" "x_step" "" }}">{{ template "login-button-label--reset-password-success-page" }}</a>
```

Comment:
The generate URL should remove `x_can_back_to_login`, otherwise, this query is persisted. (due to our convention of persisting query of name starting with `x_`)